.changelog/14634.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
```

Comment:
I was debating if this should be a `bug` or a `improvement`, but since it did cause an issue for me (using `nomad operator api` in a non-TTY session would send a -1 content length, causing endpoints like `/v1/acl/bootstrap` to [attempt decode an empty string](https://github.com/hashicorp/nomad/blob/v1.3.5/command/agent/acl_endpoint.go#L143-L147)), I set it as `bug`.